templates/python2/action-{{action_name}}.py.tpl

```diff
@@ -1,29 +1,40 @@
 #!/usr/bin/env python2
 # -*- coding: utf-8 -*-
 
-import ConfigParser
+import toml
 from hermes_python.hermes import Hermes
 from hermes_python.ffi.utils import MqttOptions
 from hermes_python.ontology import *
 import io
 
 CONFIGURATION_ENCODING_FORMAT = "utf-8"
 CONFIG_INI = "config.ini"
-
-class SnipsConfigParser(ConfigParser.SafeConfigParser):
-    def to_dict(self):
-        return {section : {option_name : option for option_name, option in self.items(section)} for section in self.sections()}
-
+PLATFORM_CONFIG_TOML = '/etc/snips.toml'
 
 def read_configuration_file(configuration_file):
     try:
         with io.open(configuration_file, encoding=CONFIGURATION_ENCODING_FORMAT) as f:
-            conf_parser = SnipsConfigParser()
-            conf_parser.readfp(f)
-            return conf_parser.to_dict()
-    except (IOError, ConfigParser.Error) as e:
+            return toml.loads(f.read())
+    except (IOError, ValueError) as e:
         return dict()
 
+def mqtt_options():
+    snips_commons = read_configuration_file(PLATFORM_CONFIG_INI).get('snips-commons', {})
+    if not snips_commons:
+        snips_commons = read_configuration_file(CONFIG_INI).get('snips-commons', {})
+
+    return MqttOptions(
+        broker_address=snips_commons.get('mqtt') or 'localhost:1883',
+        username=snips_commons.get('mqtt_username') or None,
+        password=snips_commons.get('mqtt_password') or None,
+        tls_hostname=snips_commons.get('mqtt_tls_hostname') or None,
+        tls_ca_file=snips_commons.get('mqtt_tls_cafile') or None,
+        tls_ca_path=snips_commons.get('mqtt_tls_capath') or None,
+        tls_client_key=snips_commons.get('mqtt_tls_client_key') or None,
+        tls_client_cert=snips_commons.get('mqtt_tls_client_cert') or None,
+        tls_disable_root_store=snips_commons.get('mqtt_tls_disable_root_store') or False
+    )
+
```

Comment:
add auth support. When running on a different device it will read the config.ini aswell, so it is possible to add the snips-commons section there. Fixes #9

Reply:
bascially the same https://github.com/snipsco/snips-actions-templates/pull/21/files does but with an additional fallback to config.ini and it will not throw when /etc/snips.toml does not exist, but just use the defaults